LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2022 RoboCup Federation, Inc.
+   Copyright [yyyy] [name of copyright owner]
```

Comment:
Is this intentional?

Reply:
``Copyright [yyyy] [name of copyright owner]`` is the default for the Apache License V2.0, and I don't think it is intended to be changed in the LICENSE file.

Which other files do I have to change? The source code files should start with ``Copyright 2022 RoboCup Federation, Inc.`` or an appropriate author.